script/openqa-cli

```diff
@@ -22,5 +22,7 @@ use lib "$FindBin::RealBin/../lib";
 
 use OpenQA::CLI;
 
+@ARGV = () if ($ARGV[0] // '') =~ qr/^(-h|(--)?help)$/;
```

Comment:
is this a leftover and you meant to replace it with `fix_top_level_help`?

Reply:
No. I wanted to avoid a dependency to `OpenQA::Utils` here (and all other dependencies it pulls).